### PR TITLE
237 edit profile picture and name

### PR DESCRIPTION
**Pull request by mikozet**

### CODE REVIEW

File: locales/en.json [hunk 1]
```diff
@@ -114,20 +114,22 @@
...
-    "charactersExplanation": "Only basic characters and numbers (A-Z, 0-9) are allowed, no symbols or whitespace, 3-24 characters",
     "confirmationMessage": "Great! Your profile changes have been saved!",
     "errorNameTaken": "Sorry, this name is already taken!",
     "errorInvalidName": "Ups! Invalid name, try again.",
     "errorImageCapture": "Sorry, there was a problem with image capture.",
     "errorSaveChanges": "Sorry, there was a problem with saving changes.",
+    "errorCompressImage": "Sorry, there was a problem with compressing uploaded image",
     "heading": "Edit profile",
+    "informationText": "Here you can change the main information of your Wallet.",
+    "informationText2": "Your username can have basic characters and numbers (A-Z, 0-9) are allowed, no symbols or whitespace, 3-24 characters. No symbols or whitespace are allowed.",
```
Comment (llunaCreixent) [inline locales/en.json]: I would say...
"Your username can have basic characters and numbers (a-z, A-Z, 0-9), and 3-24 characters. No symbols or white-spaces are allowed."

**Review by llunaCreixent — COMMENTED** [review]
It doesn't show the new username after successfully edit it. I need to reload

**Comment by llunaCreixent** [comment]
> It doesn't show the new username after successfully edit it. I need to reload

Also, When I edit my profile successfully, then I am redirected to my profile but it shows my old name, and then I click to edit profile again... what I see in the Edit view is my old username still.

**Comment by llunaCreixent** [comment]
Thanks for fixing the problem with the cache 🥇

File: locales/en.json (hunk 1 above)
Comment (mikozet) [reply locales/en.json]: I correct it

**Comment by mikozet** [comment]
> Also, When I edit my profile successfully, then I am redirected to my profile but it shows my old name, and then I click to edit profile again... what I see in the Edit view is my old username still.

Thx for noticing it! I corrected it

**Comment by mikozet** [comment]
> Also, When I edit my profile successfully, then I am redirected to my profile but it shows my old name, and then I click to edit profile again... what I see in the Edit view is my old username still.

**Review by llunaCreixent — APPROVED** [review]
Now it's perfect ❤️

**Pull request merged**